src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -114,48 +119,52 @@ export default {
       type: Boolean,
     },
   },
-  emits: ['update:modelValue'],
-  data() {
-    return {
-      uid: getUid(),
-      inputIsFocused: false,
-    };
-  },
-  computed: {
-    id() {
-      return `ec-checkbox-${this.uid}`;
-    },
-    errorId() {
-      return this.isInvalid ? `ec-checkbox-error-${this.uid}` : null;
-    },
-    isInvalid() {
-      return !!this.errorMessage || !!this.$slots['error-message'];
-    },
-    inputModel: {
-      get() {
-        return this.modelValue;
-      },
-      set(checked) {
-        this.$emit('update:modelValue', checked);
-      },
-    },
-  },
-  watch: {
-    indeterminate(newValue) {
-      this.updateIndeterminate(newValue);
-    },
+);
+
+const uid = ref(getUid());
+const slots = useSlots();
+
+const id = computed(() => (`ec-checkbox-${uid.value}`));
```

Comment:
Similar case for this one. This doesn't have to be a computed prop. We can just make it static too:
```js
const id = `ec-checkbox-${uid}`;
```